client/my-sites/site-settings/composing/after-the-deadline.jsx

```diff
@@ -0,0 +1,218 @@
+					{ translate( 'Automatic Language Detection' ) }
+				</FormLegend>
+				<FormSettingExplanation>
+					{ translate( 'The proofreader supports English, French, German, Portuguese and Spanish.' ) }
+				</FormSettingExplanation>
+
+				{
+					this.renderToggle( 'guess_lang', ! afterTheDeadlineModuleActive, translate(
+						'Use automatically detected language to proofread posts and pages'
+					) )
+				}
+			</div>
+		);
+	}
+
+	renderEnglishOptionsSection() {
+		const { afterTheDeadlineModuleActive, translate } = this.props;
+
+		return (
+			<div className="composing__module-settings site-settings__child-settings">
+				<FormLegend>
+					{ translate( 'English Options' ) }
+				</FormLegend>
+				<FormSettingExplanation>
+					{ translate( 'Enable proofreading for the following grammar and style rules when writing posts and pages:' ) }
+				</FormSettingExplanation>
+
+				{ this.renderToggle( 'Bias Language', ! afterTheDeadlineModuleActive, translate( 'Bias Language' ) ) }
+				{ this.renderToggle( 'Cliches', ! afterTheDeadlineModuleActive, translate( 'Clichés' ) ) }
+				{ this.renderToggle( 'Complex Expression', ! afterTheDeadlineModuleActive, translate( 'Complex Phrases' ) ) }
+				{ this.renderToggle( 'Diacritical Marks', ! afterTheDeadlineModuleActive, translate( 'Diacritical Marks' ) ) }
+				{ this.renderToggle( 'Double Negative', ! afterTheDeadlineModuleActive, translate( 'Double Negatives' ) ) }
+				{ this.renderToggle( 'Hidden Verbs', ! afterTheDeadlineModuleActive, translate( 'Hidden Verbs' ) ) }
+				{ this.renderToggle( 'Jargon Language', ! afterTheDeadlineModuleActive, translate( 'Jargon' ) ) }
+				{ this.renderToggle( 'Passive voice', ! afterTheDeadlineModuleActive, translate( 'Passive Voice' ) ) }
+				{ this.renderToggle( 'Phrases to Avoid', ! afterTheDeadlineModuleActive, translate( 'Phrases to Avoid' ) ) }
+				{ this.renderToggle( 'Redundant Expression', ! afterTheDeadlineModuleActive, translate( 'Redundant Phrases' ) ) }
+			</div>
+		);
+	}
+
+	renderIgnoredPhrasesSection() {
+		const { afterTheDeadlineModuleActive, fields, translate } = this.props;
+		const ignoredPhrases = fields.ignored_phrases
+			? fields.ignored_phrases.split( ',' )
+			: [];
+
+		return (
+			<div className="composing__module-settings site-settings__child-settings">
+				<FormLegend>
+					{ translate( 'Ignored Phrases' ) }
+				</FormLegend>
+
+				<TokenField
+					onChange={ this.onChangeIgnoredPhrases }
+					value={ ignoredPhrases }
+					disabled={ ! afterTheDeadlineModuleActive }
+				/>
+			</div>
+		);
+	}
+
+	render() {
+		const {
+			isRequestingSettings,
+			isSavingSettings,
+			selectedSiteId,
+			translate
+		} = this.props;
+
+		return (
+			<FormFieldset>
+				<div className="composing__info-link-container">
+					<InfoPopover position={ 'left' }>
+						<ExternalLink href={ 'https://jetpack.com/support/spelling-and-grammar/' } target="_blank">
+							{ translate( 'Learn more about After the Deadline' ) }
```

Comment:
I'm not quite sure if this should read `Learn more about After the Deadline` or `Learn more about Spelling and Grammar`. Although the latter would sound like we're trying to school the user: P. This could be the topic for another PR though.